content/v2.0/router/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Router: Overview"
+order: 10
+---
+
+# Overview
+
+Hanami applications use [Hanami::Router](https://github.com/hanami/router) for routing: a Rack compatible, lightweight and fast HTTP router for Ruby.
+
+## The first route
+
+With your favorite editor open `/config/routes.rb` and add the following line inside of the `main` slice block.
+
+```ruby
+get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami!']] }
+```
+
+So the whole file would look like this:
+
+```ruby
+# /config/routes.rb
+
+Hanami.application.routes do
+  slice :main, at: "/" do
+    root to: "home.show"
+
+    get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami']] }
+  end
+end
+```
+
+Then start the server with `puma config.ru` and visit [http://localhost:3000/hello](http://localhost:2300/hello).
+You should see `Hello from Hanami!` in your browser.
+
+<p><img src="/v2.0/router/hello-from-hanami.png" alt="Hanami welcome page" class="img-responsive"></p>
```

Comment:
I don't think this screenshot is particularly worth including here. It's just rendering plain-text so there's no visual elements. The browser chrome will inevitably be different throughout the guides, with different contributors using their own so minimizing then number of times we show browser screenshots is worth doing, I think :)

Reply:
I see it as a placeholder for now. The 1.3 Hanami new app had a pre-styled generated home page, which was actually worth being presented on the screenshot. I'd rather suggest changing it later to the default, instead of completely removing it.

Reply:
Agree it's useful for the Getting Started guide, but this is just a simple lambda endpoint. 

Even with [the pre-styled default page that we have in the existing Getting Started guide](https://guides.hanamirb.org/v1.3/introduction/welcome-page.png) implemented, this simple lambda endpoint will never render a view or a template, it's just returning text.

Reply:
Yeah, agree:)